explainer.md

```diff
@@ -26,13 +27,25 @@ The following web application declares in its manifest that it can handle CSV an
           "action": "/open-csv",
           "accept": {
             "text/csv": [ ".csv" ]
-          }
+          },
+          "icons": [
+            {
+              "src": “https://*/csv-file.png”,
```

Comment:
Change the curvy quotes to straight quotes so this is valid JSON.

Reply:
Also do you intend the `*` to be part of the syntax? If you're just meaning "this site", just write `"/csv-file.png"` which would be valid in a real manifest.

Reply:
I think (based on the conversation [here](https://docs.google.com/document/d/1OAkCvMwTVAf5KuHHDgAeCA3YwcTg_XmujZ7ENYq01ws/edit?disco=AAAAHcE6E5U)) that this is intended to be a list for [Image Resources](https://www.w3.org/TR/image-resource/).

(also, hi Matt!)

Reply:
Thanks, I wasn't sure why this was being tagged as invalid JSON. Yes, this was intended to be a list of `ImageResource` objects. I've updated the src link to remove the `*`, since a quick look at the spec suggests that relative links should be allowed.